src/templates/pages/example/form-validation/index.js

```diff
@@ -0,0 +1,13 @@
+import { h } from 'preact';
+import ExampleLayout from '@layouts/example';
+import Code from './code';

```

Comment:
Please see comment in re variable naming:   src/templates/pages/example/form-validation/code.js